PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/AzureArtifactComboBox.java

```diff
@@ -69,14 +69,14 @@ public void setFileFilter(final Condition<? super VirtualFile> filter) {
         this.fileFilter = filter;
     }
 
-    public synchronized void refreshItems(AzureArtifactType defaultArtifactType, String artifactIdentifier) {
+    public synchronized void refreshItems(AzureArtifact defaultArtifact) {
```

Comment:
seems can be simplified by reusing super's `setValue` method, 
let's consider refactor this part in next release.